packages/driver/cypress/e2e/cypress/stack_utils.cy.js

```diff
@@ -387,7 +386,7 @@ Error: spec iframe stack
         line: 1,
         originalFile: undefined,
         relativeFile: undefined,
-        whitespace: '    ',
+        whitespace: '            ',
       })
```

Comment:
I'm not sure why these tests had `stripIndent` in the first place. With the exception of one, they all succeeded without it. In the exception, I updated to use the correct value for whitespace